.storybook/preview.ts

```diff
@@ -0,0 +1,51 @@
+
+import { withThemeFromJSXProvider } from '@storybook/addon-styling';
+import { ThemeProvider } from 'styled-components';
+import { themes } from '../packages/extension-ui/src/components/themes'
+import { BodyTheme } from '../packages/extension-ui/src/components/View'
+
+/** @type { import('@storybook/react').Preview } */
+const preview = {
+  parameters: {
+    actions: { argTypesRegex: '^on[A-Z].*' },
+    controls: {
+      matchers: {
+       color: /(background|color)$/i,
+       date: /Date$/,
+      },
+    },
+    viewport: {
+      defaultViewport: "narrow",
+      viewports: {
```

Comment:
Should we add some more viewports to choose from? Well, at least the _desktop_ one (the default one) for convenient components viewing?

Reply:
there is a responsive view always available, I think, but I'll add desktop too, if you want

Reply:
> there is a responsive view always available

Oh ok, I haven't checked tbh. If that's the case, then we're good ✔️